### PR TITLE
 GreyTide Private Milita Clothing

### PR DESCRIPTION
**Pull request by terezi4real**
## About the PR

Adds GreyTide Militia clothes, a whole lot of em.

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added GreyTide Clothes

### CODE REVIEW

**Comment by terezi4real** [comment]
bleh i hate bugfixing

**Comment by terezi4real** [comment]
ok this is fucked dont merge a bunch of sprite errors im discovering and some missing prototypes bleh this smells, if you want you can pick thru and merge what you want, cause i am not gonna be able to work this weekend

**Comment by terezi4real** [comment]
ready if checks pass 👍

File: Resources/Prototypes/_RedEclipse/Entities/Clothing/Back/greytide.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+- type: entity
+  parent: ClothingBackpack
+  id: ClothingBackpackGreytide
+  name: greytide backpack
+  description: It's a very grey backpack.
+  components:
+  - type: Sprite
+    sprite: _RedEclipseClothing/Back/Backpacks/greytide.rsi
```
Comment (sTiKyt) [inline Resources/Prototypes/_RedEclipse/Entities/Clothing/Back/greytide.yml]: Should comment those are # TODO or # PLACEHOLDER so we don't miss it later

File: Resources/Prototypes/_RedEclipse/Entities/Clothing/Back/greytide.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+- type: entity
+  parent: ClothingBackpackSatchel
+  id: ClothingBackpackSatchelGreytide
+  name: greytide satchel
+  description: A robust satchel for violence related needs.
+  components:
+  - type: Sprite
+    sprite: _RedEclipseClothing/Back/Backpacks/greytide.rsi
```
Comment (sTiKyt) [inline Resources/Prototypes/_RedEclipse/Entities/Clothing/Back/greytide.yml]: Should comment those are # TODO or # PLACEHOLDER so we don't miss it later

File: Resources/Prototypes/_RedEclipse/Entities/Clothing/Back/greytide.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+- type: entity
+  parent: ClothingBackpackGreytide
+  id: ClothingBackpackDuffelGreytide
+  name: greytide duffel bag
+  description: A large duffel bag for holding extra greytide related goods.
+  components:
+  - type: Sprite
+    sprite: _RedEclipseClothing/Back/Backpacks/greytide.rsi
```
Comment (sTiKyt) [inline Resources/Prototypes/_RedEclipse/Entities/Clothing/Back/greytide.yml]: Should comment those are # TODO or # PLACEHOLDER so we don't miss it later

**Comment by sTiKyt** [comment]
Look, this has a problem to it, if you don't wanna write detailed commits gotta explains your changes somehow anyway

Make a list of things changed, list of things that are using placeholder data/textures, list of things you added from scratch, etc. and edit the main post at the top to include it

Also add a list of stuff i need to implement (add as roles/items/whatever)

Cuz going through all of those files and figuring it out without explanation would take A LOT of time

**Review by sTiKyt — CHANGES_REQUESTED** [review]
There seems to be a significant amount of missing information, along with commit names that aren't very intuitive.

For instance, the commit labeled as "Minor changes" ([minor changes :clueless:](https://github.com/Operation-Eclipse/red-eclipse-14/pull/22/commits/2b49a97560384f6f38c9507338b0995f722910a4)) added 1914 characters, which can hardly be considered minor; it's actually quite substantial.

Additionally, the changelog is mostly absent.

Nonetheless, I must commend your work on the proper formatting of the files and the appealing appearance of the sprites.

I understand that it would be quite an extensive effort to thoroughly comment and review everything, especially with the issues mentioned above. It might test anyone's patience, including yours. However, I don't want all of this effort to go to waste, so I'm inclined to take on the task of fixing it myself, one more time.

**Pull request closed without merging**